experimental/generation/generator/packages/library/src/mergeAssets.ts

```diff
@@ -739,3 +780,58 @@ async function parseSchemas(schemaName: string, oldPath: string, newPath: string
     await copySingleFile(newPath, mergedPath, schemaName + '.json', newFileList, feedback)
     return {oldPropertySet, newPropertySet}
 }
+
+/**
+ * @description: Get the synonyms of the enum entity.
+ * @param schemaName Name of the .schema file.
+ * @param newPath Path to the folder of the new asset.
+ */
+ async function getSynonyms(schemaName: string, newPath: string, locale: string): Promise<Map<string, Set<string>>> {
+    let template = await fs.readFile(ppath.join(newPath, schemaName + '.json'), 'utf8')
+    let newObj = JSON.parse(template)
+    let propertyValueSynonyms = new Map<string, Set<string>>()
+
+    if (newObj['$examples'] !== undefined) {
+        let globalExamples
+        if (locale in newObj['$examples']) {
+            globalExamples = newObj['$examples'][locale]
```

Comment:
Make use of null operator and properties should be before global, i.e. 
const propExamples = newObj['properties'][property]['$examples']
const globalExamples = newObj['$examples']
const examples = propExamples?.[locale] ?? propExamples?.[''] ?? globalExamples?.[locale] ?? globalExamples?.['']
const valueExamples = examples?.[`${property}Value`]